### PR TITLE
Add values to codelist options

### PR DESCRIPTION
**Pull request by lagartoverde**
This PR adds a value property to the codelist options, so they can have a label different to its value, and then it sets this value to the label for all existing codelists

### CODE REVIEW

**Comment by x-m-el** [comment]
I am not 100% certain what I say is correct, but my interpretation:

A `code-list` is a `conceptScheme` and a `conceptScheme` has many `skos-concept`s. It is these `skos-concept`s that are the "options" of a codelist.
At this point an `skos-concept` only gets used for this (and for the type of a `code-list`). 
However, this `skos-concept` feels like a dirty class that just gets molded to whatever it needs to be, making the use of SKOS bring more confusion.

- `skos-concept` has a `schema:position` property, which only makes sense for ordered list. But skos-concept is also used as a type (with cardinality=one), where this property is not needed.
- `skos-concept` is currently only used as the options of a codelist. However, the options now need some extra data (what this PR provides), which has to be band-aided into `skos-concept`, making `skos-concept` even broader, decreasing its reuseability (I am assuming this is the reason it is a separate class?)

I feel like adding this new extra ext:value to `skos-concept` is not the way to go, but the resources are structured so that it is not really possible differently, except by changing the structure.
I'm guessing we'd rather make this an easy change (like proposed in this PR), instead of trying to fix/think about the underlying issue there might be.

**Actual review of the PR:**
- changing skos:prefLabel to be the new 'label' of options (as defined in the issue) is logical in my eyes and seems to follow the usage meant by SKOS for `prefLabel`, as far as I can interpret from the SKOS docs.
- putting the data somewhere else makes sense. `ext:value` feels very broad (and not really descriptive), but as `skos-concept` is used pretty much as a "list of items", giving it a general value (without specifying what it might be) makes sense.
- in a sense, the `ext:value` should be able to be anything (so `skos-concept` can be used for any type of resource, not just a string resource), but that's just being pedantic and would increase the complexity without any gain. So just making `ext:value` a string is okay. But I wanted to at least mention it :) 

- **important actionable note**: labels are not mandatory (see comments of issue). The exisiting codelists will now end up with the label being the same as the value. This can become a problem if we'd want to limit the label in some way (e.g. text length). As they are not mandatory, it makes sens to instead:
  1) set ext:value to the value of now skos:prefLabel (like you already did)
  2) make the skos:prefLabel an empty string. (users can add labels for it if they want to)
  
  
As this is my issue, I can change the above, but as you'd need to review it anyway, asking it in text is the same as writing code that would get reviewed anyway :shrug:

**Comment by lagartoverde** [comment]
@x-m-el Sorry for the late answer, I agree with what you said, jus take into account that with this changes you will need to modify the frontend and also the plugin that consumes the codelists.
Also thinking about this, we will need to make the same changes to app-mow as the codelists should have the same structure

**Comment by x-m-el** [comment]
> @x-m-el Sorry for the late answer, I agree with what you said, jus take into account that with this changes you will need to modify the frontend and also the plugin that consumes the codelists. Also thinking about this, we will need to make the same changes to app-mow as the codelists should have the same structure

I thought about this these labels also probably needing to be in mow.
Are MOW database and Reglementaire bijdrage the same database? So if I would set prefLabel to empty string with a migration, it will not work correctly for the codelists in MOW?

**Comment by x-m-el** [comment]
Hm, that means that the plugin could break if MOW doesn't get updated *or* the plugin doesn't get updated? Because the plugin will use our (new) backend structure that is defined with this migration.
Developers might not update the plugin instantly either, so you want to let the plugin keep working.

If I set prefLabel to an empty string, every old instance of the plugin running will break.
I'm not sure how to solve that problem cleanly. Because the plugin uses a direct sparql endpoint (iirc) we can't really catch old usages of the plugin and fix it like that. 
Either way, old instances of the plugin will not work with new codelists (that have labels) anyway. So keeping the old way work is only temporarily.

I'm honestly not sure what the best course of action would be.
Or we instantly break any app running the old plugin logic, or we migrate the data in a less clean way and only break old plugin for new codelists.

**Comment by x-m-el** [comment]
Conclusion: 
- Keep old data intacked (in prefLabel)
- Add an ext:summary field instead (that is the actual label)
- all old consumers of the db will still work (just won't have the feature of a label)

File: config/migrations/20230602080249-add-summary-label-to-codelists.sparql [hunk 1]
```diff
@@ -0,0 +1,13 @@
+PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
+PREFIX ext: <http://mu.semte.ch/vocabularies/ext/>
+INSERT {
+  GRAPH ?g {
+        ?concept ext:summary "".
+  }
+}
+WHERE {
+  GRAPH ?graph {
+    ?concept a skos:Concept;
```
Comment (lagartoverde) [inline config/migrations/20230602080249-add-summary-label-to-codelists.sparql:10]: Error here, it should be . instead of ;

Comment (lagartoverde) [reply config/migrations/20230602080249-add-summary-label-to-codelists.sparql:10]: @x-m-el

**Pull request merged**